src/main/resources/rs117/hd/lighting/lights.json

```diff
@@ -19532,7 +19532,7 @@
       1.0,
       0.12752977
     ],
-    "type": "FLICKER",
+    "type": "PULSE",
```

Comment:
I assume you didn't mean to include this change to the ghost's lighting

Reply:
Oops, I did not! Thanks for checking.